### ISSUE TITLE
Selectors with spaces

### ISSUE THREAD

**Issue by Tam** [ivopetkov/html5-dom-document-php#22]
I'm trying to query for elements using selectors from a stylesheet. The majority of the have spaces in, but when I try to query by them I get this error: `Unsupported selector (div .my-thing)`.

The docs don't mention anything about spaces, so I assume they're not supported? Is it possible to add support, or is there a workaround I could use?

**Comment by ivopetkov** [comment]
Sure, I'll add support in a couple of days.

**Comment by Tam** [comment]
Awesome, thanks! 
Do you have any plans to add advanced CSS selector support (i.e. `.my-list > li:nth-child(1n + 3)`) or is this something I'd have to handle manually?

**Comment by ivopetkov** [comment]
It would be great to add such selectors, but I'm short on time the following weeks (I'll do selectors with spaces though). Are you interested in writing the code yourself? I will help with architecture and code optimization if needed.

**Comment by Tam** [comment]
I'm certainly up for it, but I'm also super busy over the next few weeks! I'll see if I can find some time to look into it.

**Comment by AndresSepar** [comment]
@Tam  You can solve it using the querySelector in this way:
```javascript
$dom->querySelector('.parent')->querySelectorAll('.children')
```

**Comment by ivopetkov** [comment]
Just released v1.1.0 with support for selectors with spaces... and some more :)
